dracut.sh

```diff
@@ -230,7 +230,7 @@ Creates initial ramdisk images for preloading modules
                          override any other compression options.
   --squash-compressor [COMPRESSION] Specify the compressor and compressor
                          specific options used by mksquashfs if squash module
-			 is called when building the initramfs.
+                         is called when building the initramfs.
```

Comment:
I see you fixed the indent here, please fix this wrong indent in first commit...

Reply:
Currently this one is merged with the 1st commit.